src/pages/VanDetails.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import Navbar from "../components/layout/Navbar";
+import Footer from "../components/layout/Footer";
+import { FaArrowLeft } from "react-icons/fa";
+import { Link } from "react-router-dom";
+
+export default function VanDetails() {
+  const params = useParams();
+  const [van, setVan] = useState(null);
+
+  useEffect(() => {
+    fetch(`/api/vans/${params.id}`)
+      .then((res) => res.json())
+      .then((data) => setVan(data.vans));
+  }, [params.id]);
+
+  return (
+    <div className="max-w-3xl mx-auto bg-light-cream">
+      <Navbar />
+
+      {van ? (
+        <div className="p-8 flex flex-col gap-10">
+          <Link
+            to={"/"}
```

Comment:
The back button should point to the /vans page as it is the page the user is on before going on the individual van page.